coffeecard/CoffeeCard.Models/DataTransferObjects/v2/Product/AddProductRequest.cs

```diff
@@ -0,0 +1,26 @@
+using System.ComponentModel;
+using System.ComponentModel.DataAnnotations;
+
+namespace CoffeeCard.Models.DataTransferObjects.v2.Product
+{
+    /// <summary>
+    /// Initiate a new product add request
+    /// </summary>
+    public class AddProductRequest
```

Comment:
For Swagger/NSwag to provide an example of a request, please also provide code comments
```xml
<example>..</example>
```

Reply:
I provided code comments but do we really need the examples? It is harder to test it with automatic examples

Reply:
I'm not sure I understand what you mean with automatic tests. However to give you some more context for my ask, then Swagger is using the examples code documentation block to give a user friendly example to the caller of an API what to expect. See for instance the v2 endpoints here https://core.prd.analogio.dk/swagger/index.html?urls.primaryName=v2#/Vouchers/Vouchers_IssueVouchers